mcs/class/System.Data/ReferenceSources/SR.cs

```diff
@@ -0,0 +1,1429 @@
+	static partial class SR
```

Comment:
We have a tool in mcs/tools/resx2sr that you can use to turn the Strings.resx from corefx into SR.cs, it outputs a simpler format than this one.